ui/shared/status/StatusChatInput.qml

```diff
@@ -215,7 +216,7 @@ Rectangle {
         insertInTextInput(messageInputField.selectionEnd, wrapWith);
 
         messageInputField.deselect()
-        formatInputMessage()
+        //formatInputMessage()
```

Comment:
Could we rather than commenting code, just remove the code that is dead?
We can still recover it via git if needed

Reply:
@alaibe of course! As this is still a WIP I have pending the clean up as soon as people confirm that it's actually working! But you are right, I will just remove those since now. Thanks!